### PR TITLE
Add NusaParagraph Emotion Classification

### PR DESCRIPTION
**Pull request by gentaiscool**
NusaParagraphEmotionClassification is a multi-class emotion classification on 10 Indonesian languages.

## Checklist

- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 

### Adding datasets checklist

**Reason for dataset addition**: ... 

- [x] I have run the following models on the task (adding the results to the pr). These can be run using the `mteb -m {model_name} -t {task_name}` command.
  - [x] `sentence-transformers/paraphrase-multilingual-MiniLM-L12-v2`
  - [x] `intfloat/multilingual-e5-small`
- [x] I have checked that the performance is neither trivial (both models gain close to perfect scores) nor random (both models gain close to random scores).
- [x] If the dataset is too big (e.g. >2048 examples), considering using `self.stratified_subsampling() under dataset_transform()`
- [x] I have filled out the metadata object in the dataset file (find documentation on it [here](https://github.com/embeddings-benchmark/mteb/blob/main/docs/adding_a_dataset.md#2-creating-the-metadata-object)).
- [x] Run tests locally to make sure nothing is broken using `make test`. 
- [x] Run the formatter to format the code using `make lint`. 

cc @Muennighoff @KennethEnevoldsen

### CODE REVIEW

File: mteb/tasks/Classification/multilingual/NusaParagraphEmotionClassification.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        name="NusaParagraphEmotionClassification",
+        dataset={
+            "path": "gentaiscool/nusaparagraph_emot",
+            "revision": "c61e8c3ee47d2dce296e9601195916b54c21d575",
+        },
+        description="NusaParagraphEmotionClassification is a multi-class emotion classification on 10 Indonesian languages.",
+        reference="https://github.com/IndoNLP/nusa-writes",
+        category="s2s",
+        type="Classification",
+        eval_splits=["train", "validation", "test"],
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/multilingual/NusaParagraphEmotionClassification.py:32]:
```suggestion
        eval_splits=["test"],
```

File: mteb/tasks/Classification/multilingual/NusaParagraphEmotionClassification.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            month = nov,
+            year = "2023",
+            address = "Nusa Dua, Bali",
+            publisher = "Association for Computational Linguistics",
+            url = "https://aclanthology.org/2023.ijcnlp-main.60",
+            pages = "921--945",
+        }
+        """,
+        n_samples={"train": 15516, "validation": 2948, "test": 6250},
+        avg_character_length={"train": 5.51, "validation": 5.49, "test": 5.45},
```
Comment (KennethEnevoldsen) [inline mteb/tasks/Classification/multilingual/NusaParagraphEmotionClassification.py:59]: size?

Comment (gentaiscool) [reply mteb/tasks/Classification/multilingual/NusaParagraphEmotionClassification.py]: I updated the sizes.

**Comment by gentaiscool** [comment]
Thank you for the review. I addressed them. I updated the avg number of character per sample, added the results, and points. I calculated the points, so there are new languages, but it only covered by other PR https://github.com/embeddings-benchmark/mteb/pull/927. So the points are 2. Plus, I added the point for the reviewer. @KennethEnevoldsen

**Pull request merged**